Common/src/geometry_structure.cpp

```diff
@@ -4978,6 +4985,243 @@ CPhysicalGeometry::CPhysicalGeometry(CGeometry *geometry, CConfig *config) {
   
 }
 
+CPhysicalGeometry::CPhysicalGeometry(CGeometry *geometry,
+                                     CConfig *config,
```

Comment:
Is this the new constructor for CPhysicalGeometry? Is the old one still there?

Reply:
Yes, this is a new constructor for CPhysicalGeometry that overloads the original. We are keeping the original *only* for periodic cases for the time being, until we can finish that update too. We'll completely remove the original after that.